scripts/install-worker.sh

```diff
@@ -175,6 +175,11 @@ sudo chmod +x /etc/eks/containerd/pull-sandbox-image.sh
 sudo chmod +x /etc/eks/containerd/pull-image.sh
 
 sudo mkdir -p /etc/systemd/system/containerd.service.d
+cat << EOF | sudo tee /etc/systemd/system/containerd.service.d/00-runtime-slice.conf
+[Service]
+Slice=runtime.slice
+EOF
```

Comment:
We still need to add a condition around this for k8s 1.24+, otherwise this LGTM

Reply:
IMHO the advice around using the runtime.slice also applies to the Docker runtime (which is where the current installation of containerd comes from as a dependency).

Reply:
Anyway, will add a conditional if that makes you feel more comfortable.

Reply:
@cartermckinnon by that condition did you mean the slice should be configured with all versions above and including 1.24.0? Isn't that the lowest kubernetes version the AMI is built for?

Sorry I don't get the exact requirement, I'd appreciate if you'd elaborate:
* why is the condition needed
* what exact version range are we talking about where the runtime.slice should be configured for for containerd

Reply:
Edit, ah OK, the EOL for 1.23 is October 11 I see. 

@cartermckinnon Well maybe let's just wait 2 weeks before we merge this PR and then no condition is necessary anymore to merge the change since 1.23 is then EOL.

Reply:
@cartermckinnon I gues this latest announcement means you won't drop 1.23 support soon: https://aws.amazon.com/blogs/containers/amazon-eks-extended-support-for-kubernetes-versions-available-in-preview/

So I'll add the version check.

Reply:
Added the version check.